src/__tests__/index.test.js

```diff
@@ -12,6 +12,11 @@ describe('normalize', () => {
     expect(() => normalize({})).toThrow();
   });
 
+  test('cannot normalize with null input', () => {
+    const mySchema = new schema.Entity('tacos');
+    expect(() => normalize(null, mySchema)).toThrow(/null/);
```

Comment:
nit: `toThrow('Unexpected input given to normalize')`

Reply:
Just making sure it has null in the error string. Not trying to worry about the rest of the wording since there isn't an conditional in there.